### ISSUE TITLE
Generating C/rust?

### ISSUE THREAD

**Issue by spitters** [mit-plv/rupicola#9]
I understand rupicola uses bedrock2, which can produce RISC-V. Is there a way to output C or rust from Rupicola/bedrock2?

**Comment by cpitclaudel** [comment]
Yes, definitely I think: Bedrock 2 also has an unverified C backend.  We haven't wired Rupicola to work with it yet.

**Comment by spitters** [comment]
Thanks! Where is it? I've looked for ExprImp in bedrock2, but it does not seem show the C backend.

**Comment by andres-erbsen** [comment]
https://github.com/mit-plv/bedrock2/blob/master/bedrock2/src/bedrock2/ToCString.v
